mitodl/open-discussions
# Add page for channel search

#### Pre-Flight checklist


#### What are the relevant tickets?
Part of #1316 

#### What's this PR do?
Implements channel search

#### How should this be manually tested?
Run `./manage.py recreate_index` to ensure the index exists then run a search on the channel. You should be able to search the title and text for a post and only the text for a comment. You should never see any profile results and the search filter should only include options for comments or posts. Selecting 'All' should only show some combination of the two, no profile results.
